Makefile

```diff
@@ -37,7 +35,7 @@ javascript: update-npm-requirements
 sass:
 	python scripts/compile_sass.py
 
-verify-generated-files: javascript sass
+verify-generated-files:
```

Comment:
This is the change I'll be movjng over - it was added for travis testing, but travis also does a `make install` earlier in its build process, so we're just repeating work needlessly